### PR TITLE
[ES] Native review of missing `G` articles 

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

Native review of G articles listed in the `No native review` section according to osu! wiki status in spanish

I did it in 2 parts to avoid confusion or any problems that may occur

### CODE REVIEW

File: wiki/Gameplay/Hit_object/Spinner/es.md [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
 
-El número máximo de giros por minuto que se puede lograr es 477, que también es la velocidad a la que el modificador de juego [Auto](/wiki/Gameplay/Game_modifier/Auto) gira mientras que el modificador de juego [Spun Out](/wiki/Gameplay/Game_modifier/Spun_Out) gira más lento. a 287 giros por minuto.
+El número máximo de giros por minuto que se puede lograr es 477, que también es la velocidad a la que el modificador del juego [Auto](/wiki/Gameplay/Game_modifier/Auto) gira mientras que el modificador del juego [Spun Out](/wiki/Gameplay/Game_modifier/Spun_Out) gira más lento a 287 giros por minuto.
```
Comment (agatemosu) [inline wiki/Gameplay/Hit_object/Spinner/es.md:14]:
```suggestion
El número máximo de giros por minuto que se puede lograr es 477, que también es la velocidad a la que el modificador del juego [Auto](/wiki/Gameplay/Game_modifier/Auto) gira, mientras que el modificador del juego [Spun Out](/wiki/Gameplay/Game_modifier/Spun_Out) gira más lento a 287 giros por minuto.
```

**Review by agatemosu — CHANGES_REQUESTED** [review]

File: wiki/Gameplay/Hit_object/Slider/Sliderbody/es.md [hunk 2]
```diff
@@ -1,14 +1,13 @@
 ---
-no_native_review: true
 stub: true
 tags:
   - slider path
   - camino del slider
   - ruta del slider
   - cuerpo del slider
-  - deslizador
+  - slider
```
Comment (agatemosu) [inline wiki/Gameplay/Hit_object/Slider/Sliderbody/es.md:8]:
```suggestion
  - deslizador
```

File: wiki/Gameplay/Hit_object/Slider/Slider_velocity/es.md [hunk 3]
```diff
@@ -1,16 +1,15 @@
 ---
-no_native_review: true
 stub: true
 tags:
   - slider speed
   - sv
-  - deslizador
+  - velocidad del slider
 ---
```
Comment (agatemosu) [inline wiki/Gameplay/Hit_object/Slider/Slider_velocity/es.md:7]:
```suggestion
  - velocidad del slider
  - deslizador
 ---
```

File: wiki/Gameplay/Hit_object/Slider/Sliderhead/es.md [hunk 4]
```diff
@@ -1,13 +1,12 @@
 ---
-no_native_review: true
 stub: true
 tags:
   - sliderstart
   - inicio del slider
   - cabeza del slider
-  - deslizador
+  - slider
```
Comment (agatemosu) [inline wiki/Gameplay/Hit_object/Slider/Sliderhead/es.md:7]:
```suggestion
  - deslizador
```

File: wiki/Gameplay/Hit_object/Slider/Slidertail/es.md [hunk 5]
```diff
@@ -1,13 +1,12 @@
 ---
-no_native_review: true
 stub: true
 tags:
   - sliderend
   - final del slider
   - cola del slider
-  - deslizador
+  - slider
```
Comment (agatemosu) [inline wiki/Gameplay/Hit_object/Slider/Slidertail/es.md:7]:
```suggestion
  - deslizador
```

**Review by agatemosu — COMMENTED** [review]
tags, this Was for who put the stable term

File: wiki/Gameplay/Hit_object/Slider/Slider_velocity/es.md [hunk 6]
```diff
@@ -1,16 +1,16 @@
...
-no_native_review: true
 stub: true
 tags:
   - slider speed
   - sv
+  - velocidad del slider
   - deslizador
----
+ ---
```
Comment (agatemosu) [inline wiki/Gameplay/Hit_object/Slider/Slider_velocity/es.md:8]: mi móvil 😡😡
```suggestion
---
```

**Review by agatemosu — APPROVED** [review]

**Review by Hiviexd — APPROVED** [review]

**Pull request merged**